plugins/services/src/js/utils/TaskTableUtil.js

```diff
@@ -3,6 +3,7 @@ import React from "react";
 /* eslint-enable no-unused-vars */
 import TableUtil from "#SRC/js/utils/TableUtil";
 import Util from "#SRC/js/utils/Util";
+import TaskUtil from "#PLUGINS/services/src/js/utils/TaskUtil";
```

Comment:
😢

Reply:
Hey @juliangieseke, what's wrong with that specific line?

Reply:
TaskUtil is broken, because it fetches data from the `CompositeState` - the more we use it, the severe the issue gets.

Reply:
Yeah, its not broken by its functionality, but from a design POV. Also its not a "Bug" which spreads in our codebase, but it is a (very) bad example which one could use as such - and copy it. thats why i said "more severe"

but in the end you are correct - we can refactor it all at once, thats why I approved it :)